Libraries/Components/WebView/WebView.android.js

```diff
@@ -189,11 +192,19 @@ var WebView = React.createClass({
   },
 
   onLoadingStart: function(event) {
+    var onLoadingStart = this.props.onLoadingStart;
+    if (onLoadingStart) {
```

Comment:
We tend to use this shorthand in our codebase:

    var onLoadingStart = this.props.onLoadingStart;
    onLoadingStart && onLoadingStart(event)